Python/pylifecycle.c

```diff
@@ -751,6 +752,11 @@ pycore_init_types(PyInterpreterState *interp)
     return _PyStatus_OK();
 }
 
+static const char INTERPRETER_TRAMPOLINE_CODE[] = {
+    0, 0,
+    INTERPRETER_EXIT, 0,
```

Comment:
Does this need to worry about endianness?

Reply:
Nope! The bytecode format is always opcode, then oparg, regardless of platform.